LPFilter/Source/PluginProcessor.h

```diff
@@ -61,7 +61,10 @@ class LpfilterAudioProcessor  : public AudioProcessor
 
 private:
     
-    void process (AudioSampleBuffer& buffer) noexcept;
+    void juceModulesProcess (dsp::ProcessContextReplacing<float>) noexcept;
```

Comment:
Don't you think that all process methods should have the same argument type (`AudioSampleBuffer& ioBuffer`)?

Reply:
oops! I missed that.